piet-gpu-hal/examples/shader/stack.comp

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The Apache license is incompatible with the piet-gpu shaders available under the MIT or UNLICENSE.  Consider making the licensing clear by replacing this section with

// SPDX-License-Identifier: Apache-2.0 OR MIT OR Unlicense